_distributor_init.py

```diff
@@ -22,5 +22,12 @@
     libs_path = os.path.abspath(os.path.join(os.path.dirname(__file__),
                                              '.libs'))
     if os.path.isdir(libs_path):
+        # for Python < 3.8, try to achieve brute force resolution
+        # of DLLs by moving into their storage dir; should have no
+        # effect for Python >= 3.8
+        owd = os.getcwd()
```

Comment:
This should probably have a try-finally to ensure getting back to the original working directory

Reply:
done; as you'll see in my expanded comments, we could have also looked into manipulating `PATH` instead of changing directories, but as I note there, it is not immediately obvious which approach is more secure/robust for loading DLLs